extensions/textline/actions/getMessages/config/fields.ts

```diff
@@ -0,0 +1,47 @@
+import { z, type ZodTypeAny } from 'zod'
+import { E164PhoneValidationOptionalSchema } from '@awell-health/extensions-core'
+import {
+  type Field,
+  FieldType,
+  StringType,
+} from '@awell-health/extensions-core'
+
+export const fields = {
+  phoneNumber: {
+    id: 'phoneNumber',
+    label: 'Phone number to filter',
+    type: FieldType.STRING,
+    stringType: StringType.PHONE,
+    description: 'Search for text messages sent from a specific phone number',
+    required: false,
+  },
```

Comment:
they dont have from or to, only phone number, for messages we send they dont show the phone number

Reply:
be aware that in TextLine the api key is bound to a specific phone number and agent. So that means that all the messages are only the ones that the phone number sent or received.

Reply:
@FlavioF Still have a question here: https://www.loom.com/share/51332184788047189c98d23e79faa16f